js/main.js

```diff
@@ -1,6 +1,9 @@
 import Game from './game';
 
 let game = new Game;
+
+window.game = game;
```

Comment:
Why's that ?

Reply:
![image](https://user-images.githubusercontent.com/786940/36340999-48a512b4-13df-11e8-8712-00095e301598.png)

This will make debugging blooming easy

Reply:
👍

Reply:
👍